src/ast/mod.rs

```diff
@@ -7289,6 +7310,16 @@ pub enum SqlOption {
         range_direction: Option<PartitionRangeDirection>,
         for_values: Vec<Expr>,
     },
+
+    Comment(CommentDef),
+
+    TableSpace(TablespaceOption),
```

Comment:
Can we add descriptions for these variants, also it would be good at least with the tablespace version to link to the docs where the option comes from?